rmgpy/data/kinetics/family.py

```diff
@@ -2182,3 +2182,57 @@ def getTopLevelGroups(self, root):
 
         return groupList
 
+    def mergeGroups(self, backbone, end):
```

Comment:
I actually discussed this briefly with somebody in the group. The other argument is that merging groups currently no context outside of families.

I think its fine either way and made a commit moving this to group if you prefer.